src/main/groovy/nebula/plugin/dependencylock/DependencyLockTaskConfigurer.groovy

```diff
@@ -235,9 +237,13 @@ class DependencyLockTaskConfigurer {
                     def ext = subproject.getExtensions().findByType(DependencyLockExtension)
                     if (ext != null) {
                         def configurations = lockableConfigurations(project, subproject, ext.configurationNames)
-                        configurations.collect { configuration ->
-                            project.dependencies.create(project.dependencies.project(path: subproject.path, configuration: configuration.name))
-                        }
+                        configurations
+                                .findAll { configuration ->
+                                    !nonGlobalLockConfiguration(configuration.name)
```

Comment:
Could use #contains for this
!configurationsToSkipForGlobalLock.contains(configuration)

Reply:
fixed